.github/workflows/code-quality.yml

```diff
@@ -18,22 +18,18 @@ jobs:
       # For repo checkout
       contents: read
     steps:
-      - uses: actions/checkout@v4
+      - name: Check out repository code
+        uses: actions/checkout@v4
 
       - name: Set up Python
         uses: actions/setup-python@v5
         with:
-          python-version: "3.9"
+          python-version: "3.10"
```

Comment:
**question:** Asked on Slack if this is okay [1]. Regardless though, it does seem needed for this work. `poetry install` was failing on `3.9` [2].

[1] https://climate-policy-radar.slack.com/archives/C078S46DJFP/p1721219970276019
[2] https://github.com/climatepolicyradar/azure-pdf-parser/actions/runs/9974156553/job/27561047918

Reply:
As discussed this needs to go in, the base docker image for the document parser is 3.9 so we'll have to bump that when integrating this release. 

https://github.com/climatepolicyradar/navigator-document-parser/blob/main/Dockerfile#L1

Reply:
Done in https://github.com/climatepolicyradar/navigator-document-parser/pull/135.